devtools-ci-index.yaml

```diff
@@ -0,0 +1,4360 @@
+        - timed: 'H 2,14 * * *'
+    publishers:
+        - email:
+            recipients: rhopp@redhat.com kkanova@redhat.com tdancs@redhat.com
+    scm:
+        - git:
+            url: https://{github_user}@github.com/{git_organization}/{git_repo}.git
+            branches:
+                - origin/master
+    <<: *che-functional-tests-template
+
+- job-template:
+    name: '{ci_project}-{git_repo}-prcheck-{test_url}-{cluster}'
+    triggers:
+        - github-pull-request:
+            github_hooks: '{github_hooks}'
+            status-context: 'ci.centos.org PR check on {test_url}'
+            <<: *github_pull_request_defaults
+    <<: *che-functional-tests-template
+
+- job-template:
+    name: '{ci_project}-{git_repo}-prcheck-build-dep'
+    triggers:
+        - build-result:
+            cron: '* * * * *'
+            groups:
+              - jobs:
+                    - 'devtools-rh-che-build-che-credentials-master'
+                results:
+                    - success
+    <<: *rh-che-automation-depbuild-template
+
+- job-template:
+    name: '{ci_project}-{git_repo}-after-rh-che-build-{test_url}'
+    triggers:
+        - build-result:
+            cron: '* * * * *'
+            groups:
+              - jobs:
+                    - 'devtools-rh-che-build-che-credentials-master'
+                results:
+                    - success
+                    - failure
+    scm:
+        - git:
+            url: https://{github_user}@github.com/{git_organization}/{git_repo}.git
+            branches:
+                - origin/master
+    beforeGetNode: |
+        #Wait for correct rh-che version to be deployed & running on prod-preview
+        isPodRunning () (
+            echo "Getting status of pod with image tag $IMAGE_TAG"
+            STATUS=`oc --server="$KUBE_SERVER" --token="$KUBE_TOKEN" get pod -l app=rhche -o json -n dsaas-preview | jq -r ".items[] | select (.spec.containers[0].image | contains(\"$IMAGE_TAG\")).status.phase"`
+            echo "Status is $STATUS"
+            if [ $STATUS = "Running" ]; then
+                return 0
+            fi
+            return 1
+        )
+        IMAGE_TAG=$(git ls-remote https://github.com/redhat-developer/rh-che.git master|cut -c1-7) #get image tag
+        TIMEOUT=60 #wait for ~ two minutes
+        CURRENT_TIME=0
+        while true
+        do
+            if [ $CURRENT_TIME -ge $TIMEOUT ]; then
+                echo "Timeout reached"
+                exit 1
+            fi
+            if isPodRunning; then
+                break
+            fi
+            echo "Pod with IMAGE_TAG=$IMAGE_TAG is not running. CURRENT_TIME=$CURRENT_TIME"
+            sleep 5
+            CURRENT_TIME=$((CURRENT_TIME+5))
+        done
+    <<: *che-functional-tests-template
+
+- job-template: &job_template_end_to_end_tests_defaults
+    name: 'devtools-test-end-to-end-{test_url}-{test_suite}'
+    defaults: global
+    node: devtools
+    after: fabric8io-fabric8-test-build-ma
```

Comment:
Our repo name is not `worker` :(, you're pushing to `f8a-golang-insights` right?